elements/osg/install.d/01-docker

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+if [ ${DIB_DEBUG_TRACE:-0} -gt 0 ]; then
+    set -x
+fi
+set -eu
+set -o pipefail
+
+# set up the repository
+sudo yum install -y yum-utils
```

Comment:
with CentOS 8 this is `dnf`, do we want to start using this or should this retain CentOS 7 compatibility?

Reply:
This builder is supposed to work for both centos7 and 8. I could do a check and use yum for centos 7 and dnf for centos8.